project/.github/ISSUE_TEMPLATE.md

```diff
@@ -12,21 +12,20 @@
 
 #### Sonata packages
 
-```bash
-$ composer show sonata-project/*
-# Put the result here.
+```
+$ composer outdated --all 'sonata-project/*'
+# Put the result here if there is one, otherwise, just remove this paragraph.
 ```
 
 #### Symfony packages
 
-```bash
-$ composer show symfony/*
-# Put the result here.
+```
+$ composer outdated --all 'symfony/*'
 ```
 
 #### PHP version
 
-```bash
```

Comment:
>  Don't forget, in the end, we will get the output of the composer, which is certainly not a bash script inside this fenced block.

First:

```bash
$ ./docker-console composer show --latest 'sonata-project/*'
sonata-project/admin-bundle              3.10.3 3.10.3 The missing Symfony Admin Generator
sonata-project/block-bundle              3.2.0  3.2.0  Symfony SonataBlockBundle
sonata-project/cache                     1.0.7  1.0.7  Cache library
sonata-project/core-bundle               3.1.2  3.1.2  Symfony SonataCoreBundle
sonata-project/doctrine-extensions       1.0.2  1.0.2  Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.1.2  3.1.2  Symfony Sonata / Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/easy-extends-bundle       2.1.10 2.1.10 Symfony SonataEasyExtendsBundle
sonata-project/exporter                  1.7.0  1.7.0  Lightweight Exporter library
sonata-project/intl-bundle               2.2.4  2.2.4  Symfony SonataIntlBundle
```

This works great, and this is a bash command, I really don't see the issue.

Second, you're talking about composer, but you modified the `php -v` command. :stuck_out_tongue: